### ISSUE TITLE
is cross branch can do cross compilation?

### ISSUE THREAD

**Issue by openedev** [stevengj/hdf5#2]
@stevengj Thanks for your work, I just posted this query on your PR [WIP](https://github.com/stevengj/hdf5/pull/1#commitcomment-37554507). Look like we have few configuration issues, may I know is it possible to do cross compile or still anything missing?

**Comment by stevengj** [comment]
No, I never completed this branch; it was mainly to help with building binaries for Julia, but the Julia developers found another solution (https://github.com/JuliaPackaging/Yggdrasil/blob/master/H/HDF5/build_tarballs.jl).

**Comment by gansm255** [comment]
Hi,
Can I use this repository for development for aarch64 with linux OS. I ran cmake and i get lot of windows related libraries missing. where i can get the source tree for aarch64 processor with linux as OS.
Please guide me. i downloaded hdf5 1.12 and other versions. but cross compile fail with error"H5detect can not execute binary file and error H5Tinit.c:. I am stuck up. all other opencv libraries are built for my platform. now left with hdf5 lib building.
please spare your time and guide me. 
Regards,
Ganesh
